data-table/table.js

```diff
@@ -157,42 +151,45 @@ function addColumns(numberOfColumns, index) {
      */
 }
 
-function createRowEntryBox() {
-    /**
-     * Creates the HTML element that will allow the user to manually
-     * affect the rows
-     * TODO: Fill this out
-     * TODO: What other methods will this need?
-     */
-}
-
 /**
  * Adds rows to the table.
  * @param {object} config       - Table configuration object
  * @param {Number} numberOfRows - Number of rows to be added
  * @param {Number} index        - Vertical index where to start adding rows
  * @returns {undefined}         - Doesn't return anything
  */
-function addRows(config, numberOfRows, index) {
+function addMultipleRows(config, numberOfRows, index) {
     // For each of the newly requested rows
     for (let newRow = 0; newRow < numberOfRows; newRow++) {
-        addSingleRow(config, index + newRow)
+        addSingleRow(config, index + newRow, undefined)
```

Comment:
Generally, undefined should be used by leaving it out entirely.
`null`: when you want to explicitly say "no value"
`undefined`: when you really haven't defined it

A general rule of thumb, you can test against `undefined` but never pass in `undefined` as a value.